src/org/opendatakit/briefcase/operations/Common.java

```diff
@@ -0,0 +1,41 @@
+package org.opendatakit.briefcase.operations;
```

Comment:
Since you’ve created new packages, it might be good to have JavaDoc for them. I don’t know if these old instructions are still current: https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#packagecomment